recipes/clinod/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eu -o pipefail
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cp -R * $outdir/
+cp $RECIPE_DIR/clinod.py $outdir/clinod
+ls -l $outdir
+ln -s $outdir/clinod $PREFIX/bin
+chmod 0755 "${PREFIX}/bin/clinod"
+
+# clinod expects the batchman binary from SNNS in its folder:
+ln -s "`which batchman`" $outdir/batchman
```

Comment:
So you're suggesting this:

```bash
ln -s $PREFIX/bin/batchman $outdir/batchman
```

Perhaps supplemented with a test like to verify the new symlink works:

```bash
$outdir/batchman -h 2>&1 | grep SNNS
```